src/applications/vaos/routes.jsx

```diff
@@ -7,8 +7,8 @@ import AppointmentListsPage from './containers/AppointmentListsPage';
 import TypeOfCarePage from './containers/TypeOfCarePage';
 import PendingAppointmentsPage from './containers/PendingAppointmentsPage';
 import PendingAppointmentPage from './containers/PendingAppointmentPage';
-import ConfirmedAppointmentPage from './containers/ConfirmedAppointmentPage';
 import ContactInfoPage from './containers/ContactInfoPage';
+import ConfirmedAppointmentPage from './containers/ConfirmedAppointmentsPage';
```

Comment:
I think this is bad merge change. There should be a `ConfirmedAppointmentPage` import and a `ConfirmedAppointmentsPage` import (Appointments is plural in the latter).

We should probably rename these to be more obviously different (we can create a separate issue for it). This was tripping me up, too.

Reply:
Hmm. I deleted the ConfirmedAppointmentsPage since it looked like the list was hard coded.

Reply:
For the imports, I think it should be: 
`
import ConfirmedAppointmentPage from './containers/ConfirmedAppointmentPage';
import ConfirmedAppointmentsPage from './containers/ConfirmedAppointmentsPage';
`

Reply:
Yeah, I think @narin is correct. And then the `appointments/confirmed` Route would use `ConfirmedAppointmentsPage` and the `appointments/confirmed/:id` Route would use `ConfirmedAppointmentPage`.